src/clr-angular/utils/focus/key-focus/key-focus.ts

```diff
@@ -108,7 +108,7 @@ export class ClrKeyFocus {
 
   private get currentItem() {
     if (this._current >= this.focusableItems.length) {
-      return null;
+      this._current = this.focusableItems.length - 1;
```

Comment:
Thanks for this.

I moved the logic into the `initializeFocus()` method, since changes can be handled from the `listenForItemUpdates()` method or from the `focusableItems` setter.

With this logic, if all items are removed temporarily, the getter will never be called.